### ISSUE TITLE
List of links no longer displaying correctly

### ISSUE THREAD

**Issue by jasondcrane** [ylefebvre/link-library#1]
Hi there. I've been using Link Library for years and I love it. Recently this page:

https://www.thejazzsession.com/archive/

...has started displaying as a bunch of overlapping columns rather than one long column. Any ideas how to fix it?

The shortcode I'm using is: [link-library settings=1]

Thanks!

Jason

**Comment by ylefebvre** [comment]
It looks to be like the new masonry display mode has been activated on your setup and is causing trouble. Could you export your Link Library settings and send them to me?

**Comment by jasondcrane** [comment]
Hi there, 

Any luck with this?

Thanks!

Jason

⁣-- 
Jason Crane
The Jazz Session
http://thejazzsession.com
Since 2007. 
The ORIGINAL jazz interview podcast.​

**Comment by ylefebvre** [comment]
Hi Jason, In case you did not notice, I asked you a question to get more information from you, namely asking you to export and provide your Link Library settings, but you did never reply. So nu luck on this until you can provide more details.

**Comment by jasondcrane** [comment]
I replied within an hour and sent the data. Perhaps it went to your spam folder? Here it is again. Thanks very much. 

⁣-- 
Jason Crane
The Jazz Session
http://thejazzsession.com
Since 2007. 
The ORIGINAL jazz interview podcast.​

**Comment by ylefebvre** [comment]
As you can see if you look up in the thread here, there was no reply from you.

There is no attachment showing up here on github with this latest reply from you. There was no attachment on the e-mail notification I received either. Are you replying to the e-mail with an attachment? That might not come through and you probably need to reply on github site and attach file here.

**Comment by jasondcrane** [comment]
Yes I just did what you asked and sent you the attachment. I'll go to the GitHub site and try it there. Since I didn't know I needed to do that I didn't do it.

⁣-- 
Jason Crane
The Jazz Session
http://thejazzsession.com
Since 2007. 
The ORIGINAL jazz interview podcast.​

**Comment by ylefebvre** [comment]
If Github won't let you attached the CSV file from your configuration, you can always e-mail it to me directly. ylefebvre at gmail dot com.

**Comment by jasondcrane** [comment]
Trying again via the website. 
[SettingSet1Export.csv](https://github.com/ylefebvre/link-library/files/6528404/SettingSet1Export.csv)
